online-shop/src/components/Details.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { RouteComponentProps} from 'react-router-dom';
+import products from './productArray';
+import '../App.css';
```

Comment:
Create a separate CSS source file for each component. This will help keep a more modular code, and your main App.css will be less crowded